python_modules/dagster/dagster/_core/definitions/decorators/asset_decorator.py

```diff
@@ -256,9 +256,28 @@ def asset(
     Examples:
         .. code-block:: python
 
+            # Basic usage: define an asset that depends on an upstream asset
+            @asset
+            def my_upstream_asset() -> int:
+                return 5
+
             @asset
             def my_asset(my_upstream_asset: int) -> int:
                 return my_upstream_asset + 1
+
+            # Using `output_required`: define an asset that conditionally yields a result
+            should_materialize = True
```

Comment:
can probably delete this line, just looks a bit confusing

Reply:
Guess I'll also delete the other comment then, for consistency.